aws/ec2_utils_for_test.go

```diff
@@ -10,7 +10,21 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func getVpcSubnets(t *testing.T, session *session.Session, vpcId string) []string {
+func getVpcSubnetsDistinctByAz(t *testing.T, session *session.Session, vpcId string) []string {
```

Comment:
I am a bit late, but FYI, there is a Terratest function to help you find an available regions. [You can see the docs here](https://github.com/gruntwork-io/terratest/blob/737e05c04c19e2a4f01a460f91ea24161540c772/docs/_docs/02_testing-best-practices/picking-instance-types.md).

Also, thank you for the fix!